requirements.txt

```diff
@@ -6,5 +6,7 @@ pendulum
 pydantic>=1.8.2,<1.10.0
 regex<=2022.3.2  # due to dateparser bug issue 1045
 scikit-learn>=1.0.0
-spacy>=3.1<4.0.0
+spacy>=3.1,<4.0.0
+thinc>=8.0.14
+torch
```

Comment:
This could be an extra dependency, which would dramatically limit the library's footprint for users that are only interested in the rule-based pipelines...